.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.1.5
```

Comment:
this PR has been around for a while, and since then the version of ruff in the skare3 template changed to v0.2.1.